### ISSUE TITLE
Maybe a guide on how to get started?

### ISSUE THREAD

**Issue by AstroOrbis** [SheepCommander/KnowledgeBase#25]

**Comment by Anthony2be** [comment]
What do you mean by that?

**Comment by Anthony2be** [comment]
Like get started on what

**Comment by AstroOrbis** [comment]
like a tutorial for small projects with command blocks, slowly introducing you to more and more commands

**Comment by Anthony2be** [comment]
> like a tutorial for small projects with command blocks, slowly introducing you to more and more commands

good idea but rn we are focusing on putting all of these resources onto a website https://sheepcommander.github.io

**Comment by AstroOrbis** [comment]
thanks!

**Comment by AstroOrbis** [comment]
can i help?

**Comment by Anthony2be** [comment]
> can i help?

with which part?

**Comment by SheepCommander** [comment]
Ye a guide would be a good idea

**Comment by Anthony2be** [comment]
also are you in the discord?

**Comment by AstroOrbis** [comment]
yes i am

**Comment by AstroOrbis** [comment]
and help with the website, and maybe add plugin commands too

**Comment by Anthony2be** [comment]
> and help with the website, and maybe add plugin commands too

k

**Comment by Anthony2be** [comment]
the github for the website is https://github.com/sheepcommander/sheepcommander.github.io
